lib/Util/Notices/EmailWithInvalidListNotice.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace MailPoet\Util\Notices;
+
+use MailPoet\Cron\Workers\SendingQueue\SendingQueue;
+use MailPoet\WP\Functions as WPFunctions;
+use MailPoet\WP\Notice;
+
+class EmailWithInvalidListNotice {
```

Comment:
I am not sure, but we've been calling `List` as `Segment` in the code. Should we be consistent with that?

Reply:
Good point 👍